services/movie.py

```diff
@@ -0,0 +1,44 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        ).distinct()
+    elif genres_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids
+        ).distinct()
+    elif actors_ids:
+        queryset = queryset.filter(
+            actors__id__in=actors_ids
+        ).distinct()

```

Comment:
There is no need to perform `distinct()` operation, use only two if conditions here remembering that querysets are lazy